docs/src/howto-guides/README.md

```diff
@@ -5,6 +5,7 @@
 3. [How to connect a cloud end-point?](./004_connect.md)
 4. [How to use `tedge mqtt` module?](./005_pub_sub.md)
 5. [How to test connection to cloud?](./007_test_connection.md)
-5. [How to test the cloud connection?](./007_test_connection.md)
-6. [How to configure the local mqtt port?](./008_config_local_mqtt_port.md)
-7. [How to trouble shoot device monitoring?](./009_touble_shooting_monitoring.md)
+6. [How to test the cloud connection?](./007_test_connection.md)
```

Comment:
Why are these lines different? They look the same to me.

Reply:
In markdown it doesn't matter what numbers you assign in an ordered list (rendered will handle proper numbering) and in the file it was 5. and 5. so I just corrected the numbers order (aesthetics).

It could even be:
```markdown
1. first item
1. second item
1. third item
```